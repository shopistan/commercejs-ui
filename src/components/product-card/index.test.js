import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import ProductCard from './index';

Enzyme.configure({ adapter: new Adapter() });

const title = 'Lizards';


let wrapped = shallow(<ProductCard title={ title } image={ '' } />);

describe('ProductCard', () => {
  it('should render the ProductCard Component correctly', () => {
    expect(wrapped).toMatchSnapshot();
  })
  it('renders ProductCard children', () => {
    expect(wrapped.find('h4')).not.toBeUndefined();
  })

});
