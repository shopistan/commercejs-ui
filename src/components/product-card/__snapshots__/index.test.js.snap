// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductCard should render the ProductCard Component correctly 1`] = `
<WithStyles(ForwardRef(Card))
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(CardMedia))
    className="makeStyles-media-2"
    image=""
    title="Lizards"
  />
  <WithStyles(ForwardRef(CardContent))>
    <WithStyles(ForwardRef(Typography))
      className="title"
      component="h4"
      gutterBottom={true}
      variant="h5"
    >
      Lizards
    </WithStyles(ForwardRef(Typography))>
    <WithStyles(ForwardRef(Typography))
      color="textSecondary"
      component="p"
      variant="body2"
    >
      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Culpa dolorum sit saepe accusamus magnam rem doloribus provident in est quae. Nobis, non pariatur. Dolor illo necessitatibus accusantium itaque quidem sapiente.
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(CardContent))>
  <WithStyles(ForwardRef(CardActions))>
    <WithStyles(ForwardRef(Button))
      color="primary"
      size="small"
    >
      Share
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      color="primary"
      size="small"
    >
      Learn More
    </WithStyles(ForwardRef(Button))>
  </WithStyles(ForwardRef(CardActions))>
</WithStyles(ForwardRef(Card))>
`;
